src/frontends/pytorch/src/op/aten_to.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset8.hpp"
+#include "op_table.hpp"
+
+#include "pt_framework_node.hpp"
+
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_to(NodeContext& context) {
+    int dtype_idx;
+    int non_blocking_idx;
+    int copy_idx;
+    int memory_format_idx;
+    if (context.get_input_size() == 5) {
+        // aten::to.dtype(Tensor(a) self, int dtype, bool non_blocking=False, bool copy=False, int? memory_format=None)
+        // -> (Tensor(a))
+        dtype_idx = 1;
+        non_blocking_idx = 2;
+        copy_idx = 3;
+        memory_format_idx = 4;
+    } else if (context.get_input_size() == 6) {
+        // aten::to.device(Tensor(a) self, Device device, int dtype, bool non_blocking=False, bool copy=False, int?
+        // memory_format=None) -> (Tensor(a)).
+        // Skipping "device" input.
+        dtype_idx = 2;
+        non_blocking_idx = 3;
+        copy_idx = 4;
+        memory_format_idx = 5;
+    } else {
+        FRONT_END_OP_CONVERSION_CHECK(false, "Unknown aten::to format");
+    }
+    // non_blocking_idx=True allows asynchronous data transfers to reduce the execution time
```

Comment:
This seems like internal detail for execution, if `non_blocking=True` why we cannot interpret this operation?